### PR TITLE
Add option to enable Octavia in standalone

### PR DESCRIPTION
**Pull request by weinimo**
Add option to enable Octavia a TripleO standalone
deployment.

JIRA: [OSPNET-1167](https://issues.redhat.com//browse/OSPNET-1167)

### CODE REVIEW

**Comment by weinimo** [comment]
/hold
testing

**Comment by weinimo** [comment]
/hold cancel
tested. works.

File: devsetup/scripts/standalone.sh [hunk 1]
```diff
@@ -53,6 +53,7 @@ MANILA_ENABLED=${MANILA_ENABLED:-true}
 SWIFT_REPLICATED=${SWIFT_REPLICATED:-false}
 TLSE_ENABLED=${TLSE_ENABLED:-false}
 CLOUD_DOMAIN=${CLOUD_DOMAIN:-localdomain}
+OCTAVIA_ENABLED=${OCTAVIA_ENABLED:-true}
```
Comment (beagles) [inline devsetup/scripts/standalone.sh:56]: where does this variable get used?

Comment (weinimo) [reply devsetup/scripts/standalone.sh:56]: Good question. Where does `BARBICAN_ENABLED` and `MANILA_ENABLED` get used? Nowhere. That line should have no effect. I'll remove it.

**Comment by omersch381** [comment]
Looks good to me

**Review by stuggi — APPROVED** [review]
/lgtm

**Pull request merged**